extension/js/background_page/background_page.ts

```diff
@@ -51,6 +53,8 @@ opgp.initWorker({ path: '/lib/openpgp.worker.js' });
 
   // storage related handlers
   BrowserMsg.bgAddListener('db', (r: Bm.Db) => BgHandlers.dbOperationHandler(db, r));
+  BrowserMsg.bgAddListener('session_passphrase_set', (r: Bm.StoreSessionSet) => passphraseTracker.set(r.key, r.value));
```

Comment:
Should we use `acctEmail` as part of the key here?
I'm not fully acquainted with the concept of switching between accounts.
Should we forget (or make invisible) passphrases stored in this cache for another account?

Reply:
Since keys are associated with a particular account and invisible to other accounts, the same should be done for pass phrases - visible to only to a particular account.

Reply:
Done